library/CMakeLists.txt

```diff
@@ -142,6 +142,7 @@ target_include_directories(hipblaslt
                                    $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
                                    $<BUILD_INTERFACE:${HIP_INCLUDE_DIRS}>
                                    $<BUILD_INTERFACE:${HIPBLAS_INCLUDE_DIRS}>
+                                   $<BUILD_INTERFACE:${HIPBLAS-COMMON_INCLUDE_DIRS}>
```

Comment:
Is HIPBLAS_INCLUDE_DIRS still required if this is using hipblas-common instead?

Reply:
Oh, that is a good point. This should not be needed anymore. @awhittle3 can you please remove this?

Reply:
Reference removed, builds as expected. I did not find any other references in the CMake files.